src/core/client/admin/components/SiteSearch/SiteFilterOption.tsx

```diff
@@ -1,38 +1,35 @@
 import { Localized } from "@fluent/react/compat";
 import cn from "classnames";
-import React, { FunctionComponent, useCallback } from "react";
-import { graphql } from "react-relay";
+import React, { FunctionComponent } from "react";
 
-import { withFragmentContainer } from "coral-framework/lib/relay";
 import { Button } from "coral-ui/components/v2";
 
-import { SiteFilterOption_site } from "coral-admin/__generated__/SiteFilterOption_site.graphql";
+import { SiteSearchListContainer_query } from "coral-admin/__generated__/SiteSearchListContainer_query.graphql";
 
 import styles from "./SiteFilterOption.css";
 
 interface Props {
-  site: SiteFilterOption_site | null;
-  onSelect: (id: string | null) => void;
+  site: SiteSearchListContainer_query["sites"]["edges"][0]["node"] | null;
```

Comment:
Instead of creating an indirect dependency between this an SiteSearchListContainer, use `{ name: string; id: string}` or make it into a fragment container.